src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback-suggestion.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="alert p-1 mb-1 border rounded" [style.max-width]="'780px'">
```

Comment:
Let's avoid inline styles and move it to scss instead

Reply:
I just renamed this file: [assessment-detail.component.html](https://github.com/ls1intum/Artemis/blob/07d0111e87d0ad48c19bb20c24ed7b70462967ae/src/main/webapp/app/assessment/assessment-detail/assessment-detail.component.html). However, I fixed it now regardless.